frontend/document/credocument.lua

```diff
@@ -109,22 +108,25 @@ function CreDocument:init()
     -- set fallback font face
     self._document:setStringProperty("crengine.font.fallback.face", self.fallback_font)
 
+    -- setting a default font before loading document actually do prevent
```

Comment:
Weird. Is it possible to add a couple of unit tests later to prevent regressions?

Reply:
Not sure, this happens to me only with one non-free epub (Charles Juliet - Rencontres avec Samuel Beckett.epub , for my own futur reference).
I previously fixed this on the coverbrowser plugin side (#3293), as it's only on its side that crashes happen, but it needs now to be moved in credocument as delayed loadDocument is gone. It seems to me there is no performance penalty in setting 2 times a default font (this one, + the one from docsettings later), so it's hopefully fine.